test/e2e/accounts/common.ts

```diff
@@ -0,0 +1,400 @@
+import { privateToAddress } from 'ethereumjs-util';
+import messages from '../../../app/_locales/en/messages.json';
+import FixtureBuilder from '../fixture-builder';
+import {
+  PRIVATE_KEY,
+  PRIVATE_KEY_TWO,
+  WINDOW_TITLES,
+  clickSignOnSignatureConfirmation,
+  switchToOrOpenDapp,
+  unlockWallet,
+  validateContractDetails,
+  multipleGanacheOptions,
+} from '../helpers';
+import { Driver } from '../webdriver/driver';
+import { DAPP_URL, TEST_SNAPS_SIMPLE_KEYRING_WEBSITE_URL } from '../constants';
+import { retry } from '../../../development/lib/retry';
+
+/**
+ * These are fixtures specific to Account Snap E2E tests:
+ * -- connected to Test Dapp
+ * -- two private keys with 25 ETH each
+ *
+ * @param title
+ */
+export const accountSnapFixtures = (title: string | undefined) => {
```

Comment:
Looks like we have a bunch of changes in this files that are not used anywhere?

Reply:
That was from a bad merge, i just removed them.